frontend/src/components/user/DialogChangeMail.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <dialog-form
```

Comment:
maybe this is a problem with dialog-form, but you cannot close the dialog with the x if there is no email